auth/server/src/main/java/com/seagates3/controller/PolicyController.java

```diff
@@ -143,6 +143,13 @@ public ServerResponse create() {
         LOGGER.error(requestBody.get("PolicyARN") + "Policy does not exists");
         return responseGenerator.noSuchEntity();
       }
+      if (policy.getAttachmentCount() > 0) {
+        LOGGER.error(
+            requestBody.get("PolicyARN") +
+            " cannot be deleted because it is attached to the entities.");
+        return responseGenerator.deletePolicyConflict(
+            "Cannot delete a policy attached to entities.");
```

Comment:
it should be either "can not" or "can't"

Reply:
@kupranay The error message is exactly copied from aws. So please let me know I still need to change.
<img width="659" alt="image" src="https://user-images.githubusercontent.com/64407225/146503512-edb978ce-f5da-4adc-ad31-6129cf80fed3.png">